key-script

```diff
@@ -46,20 +41,24 @@ fi
 PW="$($cryptkeyscript "$WELCOME_TEXT")"
 
 if [ "$HASH" = "1" ]; then
-	PW=$(printf %s "$PW" | sha256sum | awk '{print $1}')
+    P2=$(printf %s "$PW" | sha256sum | awk '{print $1}')
+else
+    P2="$PW"
 fi
 
-if check_yubikey_present; then
-	message "Accessing yubikey..."
-    R="$(ykchalresp -2 "$PW" 2>/dev/null || true)"
-	message "Retrieved the response from the Yubikey"
-	if [ "$CONCATENATE" = "1" ]; then
-		printf '%s' "$PW$R"
-	else
-	        printf '%s' "$R"
-	fi
+message "Accessing yubikey (please press the button if required)..."
```

Comment:
To my understanding I would *always* see this message, even if I have no yubikey plugged in, wouldn't I?

Reply:
Yes unfortunately I haven't come across a working way for checking the presence of a yubikey without potentially blocking on a touch.